tests/Bot.Builder.Community.Adapters.Tests/Bot.Builder.Community.Adapters.Twitter.Tests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
```

Comment:
This file should be removed.

Reply:
I think it's a bug or something on github, because the only PRs that this appears in the ones that xUnit were added and later removed 'cause it will be introduced in the PR #2